accelerate-llvm-native/Setup.hs

```diff
@@ -1,2 +1,64 @@
+{-# LANGUAGE CPP #-}
+
+module Main where
+
+import Distribution.PackageDescription
+import Distribution.PackageDescription.PrettyPrint
 import Distribution.Simple
-main = defaultMain
+import Distribution.Simple.BuildPaths
+import Distribution.Simple.LocalBuildInfo
+import Distribution.Simple.PackageIndex
+import Distribution.Simple.Setup
+import Distribution.Verbosity
+import qualified Distribution.InstalledPackageInfo                  as Installed
+
+#if MIN_VERSION_Cabal(2,2,0)
+import Distribution.PackageDescription.Parsec
+#else
+import Distribution.PackageDescription.Parse
+#endif
+
+import System.FilePath
+
+
+main :: IO ()
+main = defaultMainWithHooks simpleUserHooks
+  { postConf = postConfHook
+  , preBuild = readHook buildVerbosity
+  }
+  where
+    readHook :: (a -> Flag Verbosity) -> Args -> a -> IO HookedBuildInfo
```

Comment:
`Flag` is provided by both `Distribution.Simple.Setup` and `Distribution.PackageDescription`.